river/tree/hoeffding_adaptive_tree_classifier.py

```diff
@@ -37,6 +37,32 @@ class HoeffdingAdaptiveTreeClassifier(HoeffdingTreeClassifier):
     nominal_attributes
         List of Nominal attributes. If empty, then assume that all numeric attributes should
         be treated as continuous.
+    attribute_observer
```

Comment:
Same here.

Reply:
Updated in commit [d247eb7](https://github.com/creme-ml/creme/pull/379/commits/d247eb7b873f17442e35e1b1df8054ad05611a4b).